src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Date_Time_Regex_Ex2.java

```diff
@@ -0,0 +1,17 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Date_Time_Regex_Ex2 {
+    public static void main(String[] args) {
+        LocalDateTime dateObj = LocalDateTime.now(); // current date
```

Comment:
please rename to something more distinct/representative, for example: currentDateTime

Reply:
Done